examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -793,6 +797,10 @@ func (g *genDeepCopy) doSlice(t *types.Type, sw *generator.SnippetWriter) {
 			g.generateFor(ut.Elem, sw)
 			sw.Do("}\n", nil)
 		} else if uet.Kind == types.Interface {
+			// Note: do not generate code that won't compile as `DeepCopyinterface{}()` is not a valid function
+			if uet.Name.Name == "interface{}" {
+				klog.Fatalf("Hit an unsupported type %q", uet.Name.Name)
```

Comment:
can we improve the output? Not sure we have any helpful context here (we really should have something like a path through the types). Maybe `DeepCopy of interface{} is unsupported. Instead, use named interfaces with DeepCopy<named-interface> as one of the methods.`.

Reply:
Good call! Updated the PR with the proposed wording. You're right, it provides a better guidance for users hitting this issue.